composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "open-telemetry/opentelemetry",
+  "name": "open-telemetry/opentelemetry-meta",
```

Comment:
I propose this name to have same one as the github repository, but of course I will put whatever you want :smile: